binfmt/binfmt_exec.c

```diff
@@ -80,24 +80,30 @@ static int exec_internal(FAR const char *filename,
                          FAR const posix_spawn_file_actions_t *actions,
                          FAR const posix_spawnattr_t *attr, bool spawn)
 {
-  FAR struct binary_s *bin;
+  FAR struct binary_s *pbin;
```

Comment:
let's keep name as before to reduce the modification

Reply:
Done